deploy/crds/db_v1alpha1_cassandracluster_crd.yaml

```diff
@@ -10,4 +10,313 @@ spec:
+                  - memory
+                  type: object
+                requests:
+                  properties:
+                    cpu:
+                      pattern: ^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$
+                      type: string
+                    memory:
+                      pattern: ^([+-]?[0-9.]+)([eEinumkKMGTP]*[-+]?[0-9]*)$
+                      type: string
+                  required:
+                  - cpu
+                  - memory
+                  type: object
+              type: object
+            runAsUser:
+              description: RunAsUser define the id of the user to run in the Cassandra
+                image
+              format: int64
+              minimum: 1
+              type: integer
+            service:
+              properties:
+                annotations:
+                  additionalProperties:
+                    type: string
+                  description: Annotations specifies the annotations to attach to
+                    headless service the CassKop operator creates
+                  type: object
+              type: object
+            topology:
+              description: Topology to create Cassandra DC and Racks and to target
+                appropriate Kubernetes Nodes
+              properties:
+                dc:
+                  description: List of DC defined in the CassandraCluster
+                  items:
+                    properties:
+                      labels:
+                        additionalProperties:
+                          type: string
+                        description: Labels used to target Kubernetes nodes
+                        type: object
+                      name:
+                        description: Name of the CassandraDC
+                        type: string
+                      nodesPerRacks:
+                        description: 'Number of nodes to deploy for a Cassandra deployment
+                          in each Racks. Default: 1. Optional, if not filled, used
+                          value define in CassandraClusterSpec'
+                        format: int32
+                        type: integer
+                      numTokens:
+                        description: 'NumTokens : configure the CASSANDRA_NUM_TOKENS
+                          parameter which can be different for each DD'
+                        format: int32
+                        type: integer
+                      rack:
+                        description: List of Racks defined in the Cassandra DC
+                        items:
+                          properties:
+                            labels:
+                              additionalProperties:
+                                type: string
+                              description: Labels used to target Kubernetes nodes
+                              type: object
+                            name:
+                              description: Name of the Rack
```

Comment:
Please ensure that dc and rack names do not contain a dash as it is used internally as a delimiter. See #158

Reply:
I added validation regexes